spec/support/features/session_helper.rb

```diff
@@ -10,6 +10,14 @@ def sign_up_with(email)
       click_button t('forms.buttons.submit.default')
     end
 
+    def select_alternate_2fa_option(option)
```

Comment:
I know that in American English, "alternate" can also be used to mean "available as another choice", but I prefer to use "alternative", because that can only mean one thing, whereas the main definition of "alternate" is "every other".

Reply:
How about 'different'?

Reply:
How about `choose_another_security_option` to mimic the text on the screen?